File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Construction_Site_Models/Edit_Construction_Site_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Edit_Construction_Site_Data{
    session?: Session_Data;
    
    id:number;
    
    construction_site_name : string;
    address : string;
    cities_list_id_fk : number;
    postal_code : string;
    comment : string;
}

File: Managemate/ManagemateAPI/ManagemateAPI/Management/M_Construction_Site/Input_Objects/Edit_Construction_Site_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Construction_Site.Input_Objects
{
    public class Edit_Construction_Site_Data
    {
        public Session_Data session { get; set; }
        public long id { get; set; }
        public string construction_site_name { get; set; }
        public string address { get; set; }
        public long cities_list_id_fk { get; set; }
        public string postal_code { get; set; }
        public string comment { get; set; }
    }
}
